src/cephadm/cephadm.py

```diff
@@ -3138,13 +3143,14 @@ def get_container(ctx: CephadmContext,
         envs.append('TCMALLOC_MAX_TOTAL_THREAD_CACHE_BYTES=134217728')
     if container_args is None:
         container_args = []
+    if daemon_type in Ceph.daemons or daemon_type in Ceph.gateways:
```

Comment:
With all daemons and both iscsi and nfs services included, what entities managed by cephadm is this check meant to exclude?

Reply:
anything not using a ceph container. Grafana, alertmanager, node-exporter, prometheus, keepalived, haproxy, SNMP, loki, promtail, and we also support custom containers that would be excluded here. `Ceph.daemons` is only
```
('mon', 'mgr', 'osd', 'mds', 'rgw', 'rbd-mirror', 'crash', 'cephfs-mirror', 'ceph-exporter')
```